README.md

```diff
@@ -226,6 +226,9 @@ TODO: Move this code over to the idseq-dag repo.
 
 ## Release notes
 
+- 3.8
```

Comment:
Please coordinate with Emily, since she will be using 3.8 already.

Reply:
Maybe we could combine our changes?

Reply:
I think they should still be separate PRs, but just make sure you coordinate, so you do not have to change these last minute.

Reply:
Discussed this with Lucia offline, but what we'll do is merge both of these PRs into a separate branch that's a copy of master, then merge that branch into master